src/components/PageSettingsModal/PageSettingsModal.tsx

```diff
@@ -146,6 +147,23 @@ export const PageSettingsModal = ({
     })
   }
 
+  const getDefaultPermalink = (title: string) => {
+    if (!title) return ""
+    const titleSlug = title
+      .toLowerCase()
+      .replace(/[^a-z0-9]+/g, "-")
+      .replace(/^-+|-+$/g, "")
```

Comment:
i think we only need `/^-+$/g`

separately, i'm not sure why we're replacing all instances of `-` with an empty string - that seems like a mistake? (eg: `a school bus` -> `a-school-bus` -> `aschoolbus`)

Reply:
hey @seaerchin, nope we are only replacing instances - at the start or at the end of the string. 

```
> "a school bus".toLowerCase().replace(/[^a-z0-9]+/g, "-").replace(/^-+|-+$/g, "")
'a-school-bus'
```

Reply:
ah ok, my bad. so this first does a replacement of spaces to `-` then removes leading/trailing `-`.